### PR TITLE
Updates the requirements as per the feature name

### PR DESCRIPTION
**Pull request by san7ket**

### CODE REVIEW

**Comment by kbidarkar** [comment]
Feel, this is more to do with a specific region in AWS, that is Govcloud and hence needs it's own module.
@san7ket  +1

File: tests/foreman/ui/test_computeresource_aws_govcloud.py [hunk 1]
```diff
@@ -0,0 +1,463 @@
...
+            3. Provide a valid name to AWS Govcloud ec2 compute resource.
+            4. Test the connection using Load Regions.
+            5. Provide a valid custom region
+
+        :expectedresults: An AWS Govcloud ec2 compute resource is created
+            successfully.
+
+        :BZ: 1456942
+
+        :Caseautomation: Automated
```
Comment (ntkathole) [inline tests/foreman/ui/test_computeresource_aws_govcloud.py]: notautomated

File: tests/foreman/ui/test_computeresource_aws_govcloud.py [hunk 2]
```diff
@@ -0,0 +1,463 @@
...
+        :steps:
+            1. Create a compute resource of type ec2.
+            2. Provide a valid Access Key and Secret Key.
+            3. Provide a valid name to AWS Govcloud ec2 compute resource.
+            4. Test the connection using Load Regions and submit.
+
+        :expectedresults: An AWS Govcloud ec2 compute resource is created
+            successfully.
+
+        :Caseautomation: Automated
```
Comment (ntkathole) [inline tests/foreman/ui/test_computeresource_aws_govcloud.py]: notautomated

**Comment by san7ket** [comment]
@ntkathole Please review , need this to merged before next snap hits

**Pull request merged**